K2Bridge/Factories/BucketFactory.cs

```diff
@@ -46,7 +48,31 @@ public static DateHistogramBucket CreateDateHistogramBucketFromDataRow(DataRow r
                     continue;
                 }
 
-                dhb.Aggs[clmn.ColumnName.Substring(1)] = new System.Collections.Generic.List<double>() { Convert.ToDouble(row[clmn.ColumnName]) };
+                // Step 1: create new Regex.
+                Regex regex = new Regex(@"^_(\d+)%*(100\.00|[0-9]?[0-9]\.[0-9]{1})$");
```

Comment:
can you add a comment on what this regex tries to match

Reply:
Done